app/main/routes.py

```diff
@@ -77,8 +79,12 @@ def upload():
                 fund=Config.FUND_NAME,
             )
         else:
-            current_app.logger.info("Validation errors found during upload")
-
+            combined_error = {
+                key: [error[key] for error in validation_errors]
+                for key in ["cell_index", "description", "section", "sheet"]
+            }
```

Comment:
Combining the errors like this means we lose a lot of information. For instance, we don't know which cells are attributed to which types of error any more. I think it's best to just log them as they are.